src/main/java/Response/Response.java

```diff
@@ -0,0 +1,36 @@
+package Response;
+
+import java.io.*;
+
+public class Response {
+  public String res200(String content) {
+    String headers = "HTTP/1.1 200 OK\r\n" +
+      "Content-Length: " + (content + "\r\n\r\n").length() + "\r\n" +
```

Comment:
I see that the `CRLF` is present as strings in a lot of your Response / Request object code.  Maybe that's a good opportunity for a constant on a top-level object?